numpy/testing/tests/test_utils.py

```diff
@@ -232,22 +232,22 @@ def test_array_vs_scalar_strict(self):
         b = 1.
 
         with pytest.raises(AssertionError):
-            assert_array_equal(a, b, strict=True)
+            self._assert_func(a, b, strict=True)
```

Comment:
`TestEqual`, the class used to test `assert_equal`, inherits from this class, `TestArrayEqual`, which is used to test test `assert_array_equal`. 

Previously, these three tests of the `strict` parameter would run twice - once for `TestArrayEqual`, and once for `TestEqual` - and use `assert_array_equal` both times. Now, these will test the appropriate assertion function each time they run.

These were the only tests of the `strict` parameter added when the `strict` parameter was added to `assert_array_equal` in gh-21595. Assuming they were sufficient there, enabling them for `assert_equal` should be sufficient here.